src/components/WalletTransfer2.js

```diff
@@ -182,16 +181,26 @@ class WalletTransfer2 extends Component<Props> {
   transferCoin = () => {
     const { to, value, gas } = this.state
     const root = this
-
+    let setOption
     caver.klay.accounts.wallet.add(this.wallet.privateKey, this.HRADataChange())
-
-    caver.klay.sendTransaction({
+    const contractInstance = new caver.klay.Contract(krc20ABI, to)
```

Comment:
Why did you use caver.klay.Contract?
You would be better to use "caver.klay.isContractAccount(address [, defaultBlock] [, callback])"